stories/src/ui/StepByStepNav.stories.mdx

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components'
```

Comment:
It would be nice to have a few more states in storybook IMHO

Reply:
Not necessary, you can play around with the controls and set `itemType`, `activeItem` and `stepsCompleted`.

Reply:
For good measure I added two more scenarios